### ISSUE TITLE
mixing tap and doubletap on the same element not working on the latest version

### ISSUE THREAD

**Issue by equilerex** [RyanMullins/angular-hammer#35]
Using the latest version, tap and double tap can't be used at the same time on the same element,

the issue seems to be that the "type" passed into "function addRecognizer (manager, type)" is in one case an object {type:tap} instead of string "tap"

ive made an example here (see the console for errors and button actions): 
http://codepen.io/anon/pen/ByYMqx

**Comment by mattmcdonald-uk** [comment]
You can work around this bug (one way: do a typeof check on type, if object use type.type), but doing so will cause several tap events to be fired for every doubletap.

To avoid that you would need to make use of requireFailure, but that is a current issue (#23).

Perhaps fixing this should be part of the work for #24?

**Comment by equilerex** [comment]
yeah, i tried it with type.type  but since the old version was working fine, i just continued using that one til it gets a fix... just wanted to point it out if it had not already been found ;)

**Comment by RyanMullins** [comment]
Check out the fix in #36, I think that will address your issue. I'm swamped with other stuff at the moment, so this won't be published in a release until at least the weekend (21 Feb 2015).

**Comment by luohaoGit** [comment]
have your problem solved?

**Comment by kfei** [comment]
@RyanMullins Any plan on publishing next release to include #36? Thanks a lot!
